### DISCUSSION TITLE
Support for Firebase Genkit

### DISCUSSION THREAD

**Discussion by debkanchan** [langfuse/langfuse#3351] [Ideas]
### Describe the feature or potential improvement

Currently there are no good tracing solutions for Firebase Genkit. It's still not v1 but devex is miles ahead of langchain. The only thing genkit misses rn is tracing.

### Additional information

Full disclosure: I'm the author of genkitx-graph plugin

**Comment by marcklingen** [comment]
Thanks for sharing. Genkit seems to support OpenTelemetry based tracing ([docs](https://firebase.google.com/docs/genkit/plugin-authoring-telemetry)). Can you add this as another use case to the OTel discussion [here](https://github.com/orgs/langfuse/discussions/2509)?

**Reply by dmeremyanin** [reply]
Hey @marcklingen - I've opened a PR adding support for Genkit spans https://github.com/langfuse/langfuse/pull/12199. Would love your thoughts when you get a chance

**Comment by marcelfolaron** [comment]
Hi, 

I just published a (early version) langfuse plugin for js in case anyone is interested.  

https://www.npmjs.com/package/genkitx-langfuse
https://github.com/marcelfolaron/genkitx-langfuse
